tests/ops/op_math/test_composite.py

```diff
@@ -357,8 +357,8 @@ def test_overlapping_ops_property(self):
             [
                 qml.sum(qml.PauliX(0), qml.PauliY(5), qml.PauliZ(10)),
                 qml.prod(qml.PauliX(10), qml.PauliY(2)),
-                qml.Hamiltonian([1, 1], [qml.PauliX(2), qml.PauliZ(7)]),
                 qml.PauliY(7),
+                qml.Hamiltonian([1, 1], [qml.PauliX(2), qml.PauliZ(7)]),
```

Comment:
`qml.Hamiltonian([1, 1], [qml.PauliX(2), qml.PauliZ(7)])` came after `qml.PauliY(7)` in the original op, and this ordering should've been maintained in `overlapping_ops`